src/app.ts

```diff
@@ -31,7 +31,7 @@ program
   .description('Builds the report into a folder named "dist"')
   .action(() => {
     const cliConfig = loadCliConfig();
-    Builder.create().build(cliConfig.distPath, cliConfig.buildCommand).catch(err => {
+    new Builder(console).build(cliConfig.distPath, cliConfig.buildCommand).catch(err => {
```

Comment:
At some point, we might think on including a built-in console logger that includes automatically the colors, for instance:

```ts
export class Logger {
  public function success(message: string): void {
     console.log(chalk.green(message));
  }
}
```

Or also, we might think on relying into a library for this.